### PR TITLE
Add MyBinder Support

### LINKED ISSUE #12: mybinder
lets get mybinder setup so people can play with these without having to setup a software environment.

### PR DESCRIPTION
**Pull request by mikemhenry**
Just a draft for now, but testing out mybinder support

### CODE REVIEW

**Comment by mikemhenry** [comment]
Use this link to test:
https://mybinder.org/v2/gh/mikemhenry/notebook_tutorials/mikemhenry-patch-1

**Comment by mikemhenry** [comment]
Looks like the RDF example works!
Got this error for the OPV one:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-0f81a1ee2dd7> in <module>
      5 
      6     # Load GAFF
----> 7     GAFF = foyer.forcefields.load_GAFF()
      8 
      9     # Use foyer to parameterize -- this creates a parmed structure

AttributeError: module 'foyer.forcefields.forcefields' has no attribute 'load_GAFF'
```
and this one for the temp-thermostat one:

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-3-b2af4ff1a444> in <module>
----> 1 bcc = build_system()

<ipython-input-2-91cb2df90cec> in build_system(packing)
     10         system = hoomd.init.create_lattice(unitcell=(hoomd.lattice.fcc(a=1.58)), n=6)
     11     elif packing == 'bcc':
---> 12         system = hoomd.init.create_lattice(unitcell=(hoomd.lattice.bcc(a=1.29)), n=6)
     13     hoomd.dump.gsd('{}_system.gsd'.format(packing), group = hoomd.group.all(), period=None, overwrite=True)
     14     return system

/srv/conda/envs/notebook/lib/python3.7/site-packages/hoomd/init.py in create_lattice(unitcell, n)
     61                                   n=[100,58]);
     62     """
---> 63     hoomd.context._verify_init();
     64     hoomd.util.print_status_line();
     65 

/srv/conda/envs/notebook/lib/python3.7/site-packages/hoomd/context.py in _verify_init()
    394 
    395     if exec_conf is None:
--> 396         raise RuntimeError("Call context.initialize() before any method")
    397 
    398 ## \internal

RuntimeError: Call context.initialize() before any method
```
If I re-run the error goes away so we just need to change the logic a bit for the hoomd init.

**Comment by jennyfothergill** [comment]
let's add 
```
- pip:
    - git+https://github.com/rsdefever/GAFF-foyer
```
to environment.yml and give it a go C:

**Comment by mikemhenry** [comment]
Jenny mentioned the error could be solved by installing the GAFF forcefield

**Comment by mikemhenry** [comment]
Just so we have one point of truth, I was thinking we should change the readme to use this command to create the conda env:

https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file

File: README.md
Comment (jennyfothergill) [inline README.md:9]: ah so much cleaner 👯

**Review by jennyfothergill — APPROVED** [review]

**Comment by mikemhenry** [comment]
Doing another rebuild, but might have to do some troubleshooting since it looks like gafffoyer gets installed:
```
Successfully built gafffoyer
Installing collected packages: gafffoyer
Successfully installed gafffoyer-0.0
```

But still getting:
`AttributeError: module 'foyer.forcefields.forcefields' has no attribute 'load_GAFF'`

So it will require more troubleshooting, but do we want to wait to get that fixed before we merge this?

**Comment by jennyfothergill** [comment]
probably not because after chatting with Ryan today I realise I'm going to want to use antefoyer instead of GAFF-foyer to type the cc and cd atoms. 
I think the reason the mybinder installation isn't working is because of the way the foyer entrypoints work? (It only works for me locally if I do `pip install -e .` without the `-e` flag it doesn't work... Why/how has been explained to me but I forget.)

**Comment by mikemhenry** [comment]
Okay let's merge this after I update the README urls, the notebook that isn't working does have instructions that mention how  the plugin needs to be installed, I'll leave it to another PR to get that working

**Review by jennyfothergill — APPROVED** [review]

**Pull request merged**
